images/node/README.md

```diff
@@ -12,4 +12,4 @@ Roughly this image is [the base image](./../base), with the addition of:
 See [`docs/node-image`][docs/node-image.md] for more design details.
 
 [pkg/build/node_image.go]: ./../../pkg/build/node/node.go
-[docs/node-image.md]: ./../../docs/design/node-image.md
+[docs/node-image.md]: ./../../site/content/docs/design/node-image.md
```

Comment:
maybe let's link to the hosted docs instead for these site/ links. we should also probably just make this say "node-image.md" instead of docs/

Reply:
addressed.